low-level/framework/base/sspl_constants.py

```diff
@@ -252,6 +252,40 @@
 
 SSPL_CONFIGS = ['log_level', 'cli_type', 'sspl_log_file_path', 'cluster_id', 'storage_enclosure', 'setup', 'operating_system']
 
+# SSPL python 3rd party package dependencies
+pip3s_packages_main = {
+    "jsonschema": "3.2.0",
+    "pika": "1.1.0",
+    "python-daemon": "2.2.4",
+    "requests": "2.25.1",
+    "zope.component": "4.6.2",
+    "zope.event": "4.5.0",
+    "zope.interface": "5.2.0",
+    "pyinotify": "0.9.6",
+    "cryptography": "2.3"
+}
+
+pip3s_packages_test = {
+    "Flask": "1.1.1"
+}
+
+# SSPL 3rd party RPM dependencies
+rpm_dependencies = [
```

Comment:
Should have structure to maintain versions too, similar to python pkgs

Reply:
Now, added the version for each rpms.